en/components/general/localization.md

```diff
@@ -1,32 +1,87 @@
 
 With only a few lines of code, users can easily localize the strings in Ignite UI for Angular components.
 
 ## Angular Localization Example
-<div class="sample-container loading" style="height:550px">
-    <iframe id="localization-sample-1-iframe" src="{environment:demosBaseUrl}/services/localization-sample-1" width="100%" height="100%" seamless="" frameborder="0" onload="onSampleIframeContentLoaded(this);" alt="Angular Localization Example"></iframe>
+<div class="sample-container loading" style="height:800px">
+    <iframe id="localization-all-resources-iframe" src="{environment:demosBaseUrl}/services/localization-all-resources" width="100%" height="100%" seamless="" frameborder="0" onload="onSampleIframeContentLoaded(this);" alt="Angular Localize Application Example"></iframe>
 </div>
 <p style="margin: 0;padding-top: 0.5rem">Like this sample? Get access to our complete Angular toolkit and start building your own apps in minutes. <a class="no-external-icon mchNoDecorate trackCTA" target="_blank" href="https://www.infragistics.com/products/ignite-ui-angular/download" data-xd-ga-action="Download" data-xd-ga-label="Ignite UI for Angular">Download it for free.</a></p>
 <div>
-<button data-localize="codesandbox" disabled class="codesandbox-btn" data-iframe-id="localization-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on codesandbox</button>
-<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="localization-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+<button data-localize="codesandbox" disabled class="codesandbox-btn" data-iframe-id="localization-all-resources-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on codesandbox</button>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="localization-all-resources-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
 ## Usage
 
-### Localize entire application
+### Load localized resources from npm package
+
+To localize an application for one of the available in the `igniteui-angular-i18n` package languages, the most straight forward approach is to:
+
+Install the package first:
+
+`npm install igniteui-angular-i18n --save-dev`
+
+Then import the resource strings for the desired language and call the `changei18n` function passing the corresponding resource object:
+
+```typescript
+// app.component.ts
+...
+import { changei18n } from "igniteui-angular";
+import { IgxResourceStringsJA } from 'igniteui-angular-i18n';
+...
+public ngOnInit(): void {
+    ...
+    changei18n(IgxResourceStringsJA);
+    ...
+}
+```
+
+>Note: Feel free to contribute to the `igniteui-angular-i18n` package.
```

Comment:
Feel free to submit any ideas or changes related to the `igniteui-angular-i18n` package by logging them in the [repository](https://github.com/IgniteUI/igniteui-angular-i18n/)